charts/lnd/templates/wallet-init-hook.yaml

```diff
@@ -14,7 +15,7 @@ spec:
     image: "boxboat/kubectl"
     imagePullPolicy: IfNotPresent
     command: ['/bin/sh']
-    args: ['-c','apk update; apk add expect; ./walletInit.exp {{.Values.global.network}} $LND_PASS {{ include "lnd.fullname" . }}-0;']
+    args: ['-c', 'apk update; apk add expect; kubectl wait --for=condition=ready --timeout=150s pod {{ include "lnd.fullname" . }}-0; ./walletInit.exp {{.Values.global.network}} $LND_PASS {{ include "lnd.fullname" . }}-0;']
```

Comment:
the `.exp` is confusing to me. what is that supposed to mean?

Reply:
Any extension can be used, but tt's a convention for expect scripts to use `.exp` extension. More info [here](https://docs.oracle.com/cd/E35328_01/E35336/html/vmcli-script.html). 

It's a way of self documenting to make it clear that it's not a bash script.